plugins/Compressor/CompressorControlDialog.cpp

```diff
@@ -645,9 +646,12 @@ void CompressorControlDialog::resizeEvent(QResizeEvent *event)
 
 void CompressorControlDialog::wheelEvent(QWheelEvent * event)
 {
+	auto scroll = Scroll(event);
+	event->accept();
+
 	const float temp = m_dbRange;
-	const float dbRangeNew = m_dbRange - copysignf(COMP_GRID_SPACING, event->angleDelta().y());
-	m_dbRange = round(qBound(COMP_GRID_SPACING, dbRangeNew, COMP_GRID_MAX) / COMP_GRID_SPACING) * COMP_GRID_SPACING;
```

Comment:
Since `scroll.getSteps()` returns an int I don't think we need to round to the nearest multiple of `COMP_GRID_SPACING` because the new value will always be a multiple of `COMP_GRID_SPACING`.